model/src/test/scala/explore/model/ModelOpticsSuite.scala

```diff
@@ -5,16 +5,22 @@ package explore.model
 
 import explore.model.arb.all._
 import lucuma.core.math.arb.ArbDeclination
-import lucuma.core.math.arb.ArbProperMotion
 import lucuma.core.math.arb.ArbRightAscension
+import lucuma.core.model.arb.ArbSiderealTracking
+import eu.timepit.refined.scalacheck.string._
+import eu.timepit.refined.types.string._
+import eu.timepit.refined.cats._
 import monocle.law.discipline.LensTests
 import munit.DisciplineSuite
+import org.scalacheck.Cogen
 
 class ModelOpticsSuite
     extends DisciplineSuite
     with ArbDeclination
     with ArbRightAscension
-    with ArbProperMotion {
+    with ArbSiderealTracking {
+  implicit val coGenNonEmptyString: Cogen[NonEmptyString] = Cogen[String].contramap(_.value)
```

Comment:
I think there's a `refined-scalacheck` package that should provide this.

Reply:
Yes I need to use better imports